varsim.py

```diff
@@ -21,6 +23,51 @@
 REQUIRE_VARSIMJAR = not os.path.isfile(DEFAULT_VARSIMJAR)
 if REQUIRE_VARSIMJAR: DEFAULT_VARSIMJAR = None
 
+def convertCN(filenames, operation):
+    """
+    convert '2/1'-like copy number to a single number(e.g. 2)
+    0 will be considered same as 1
+    by default the max number will be kept
+    the change is in place
+    """
+    if operation != "two2one" and operation != "one2two":
+        raise ValueError("Only two2one or one2two allowed")
+    two2one = operation == "two2one"
+    delimiter = re.compile('[/|]')
+    for name in filenames:
+	with open(name, 'r') as file_fd:
+            output = tempfile.NamedTemporaryFile(mode = 'r+w', delete = False)
+            for l in file_fd:
+		l = l.rstrip()
+                fields = l.split("\t")
```

Comment:
Looks like the indentation is an extra space when compared to the previous line.